Pseudo/pseudo-classes.html

```diff
@@ -0,0 +1,41 @@
+<!DOCTYPE html>
+<html lang="en">
+	<head>
+		<meta charset="utf-8">
+		<meta name="viewport" content="width=device-width,initial-scale=1.0">
+		<title>Pseudo Classes</title>
+		<link rel="stylesheet" type="text/css" href="css/base.css">
+		<link rel="stylesheet" type="text/css" href="css/pseudoClass.css">
+	</head>
+	<body>
+	
+		<div class="container">
+
+			<!--AnchorStyling-->
+			<div class="section anchorStyling">
+				<h3>Anchor Styling</h3>
+				<a href="#">Effects</a>
+			</div>
+
+			<!--childpseudo classes-->
+			<div class="section pseudoChild">
+				<h3>Pseudo Child Classes</h3>
+				<ul>
+					<li><a href="#">ONE</a></li>
+					<li><a href="#">TWO</a></li>
+					<li><a href="#">THREE</a></li>
+					<li><a href="#">FOUR</a></li>
+				</ul>
+			</div>
+			
+			<!-- input Focus -->
+			<div class="section inputsTypes">
+				<h3>Input Focus</h3>
+				<input type="text">
+				<input type="text" autofocus="">
```

Comment:
Why should autofocus even have blank values?

Reply:
right done